src/tests/unittests/WireTests.cpp

```diff
@@ -408,6 +408,39 @@ TEST_F(WireTests, StructureOfValuesArgument) {
     FlushClient();
 }
 
+// Test that the wire is able to send structures that contain objects
+TEST_F(WireTests, StructureOfObjectsArgument) {
+    nxtBindGroupLayoutBuilder bglBuilder = nxtDeviceCreateBindGroupLayoutBuilder(device);
+    nxtBindGroupLayoutBuilderSetBindingsType(bglBuilder, NXT_SHADER_STAGE_BIT_FRAGMENT, NXT_BINDING_TYPE_SAMPLER, 0, 1);
```

Comment:
nit: for the purpose of this test, I don't think we need to call `BindGroupLayoutBuilder::SetBindingsType`: empty layouts would be ok.

Reply:
Ah, this was extra from trying to catch the bug. But I guess it wasn't necessary after all.